src/updatedialog.cpp

```diff
@@ -0,0 +1,67 @@
+#include "updatedialog.h"
+#include "ui_updatedialog.h"
+
+#include "lootdialog.h" // for MarkdownPage
+#include <QWebChannel>
+
+using namespace MOBase;
+
+UpdateDialog::UpdateDialog(QWidget* parent) :
+  QDialog(parent, Qt::WindowMaximizeButtonHint | Qt::WindowCloseButtonHint), ui(new Ui::UpdateDialog)
+{
+  // Basic UI stuff
+  ui->setupUi(this);
+  connect(ui->installButton, &QPushButton::pressed, this, [&]{ done(QDialog::Accepted); });
+  connect(ui->cancelButton, &QPushButton::pressed, this, [&]{ done(QDialog::Rejected); });
+
+  // Replace a label with an icon
+  QIcon icon = style()->standardIcon(QStyle::SP_MessageBoxQuestion);
+  QPixmap pixmap = icon.pixmap(QSize(32, 32));
+  ui->iconLabel->setPixmap(pixmap);
+  ui->iconLabel->setScaledContents(true);
+
+  // Setting up the Markdown stuff
+  auto* page = new MarkdownPage(this);
```

Comment:
Is all of this really necessary? Qt support markdown in `QTextEdit` since 5.14.

Reply:
IDK.  That's how it's handled elsewhere in MO.  So ... hooray consistency.